Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -136,6 +136,11 @@ struct Component {
     u8 hsample_factor { 1 }; // Hi, Horizontal sampling factor
     u8 vsample_factor { 1 }; // Vi, Vertical sampling factor
     u8 qtable_id { 0 };      // Tqi, Quantization table destination selector
+
+    // The JPEG specification does not specify which component correspond to
```

Comment:
s/correspond/corresponds/

Isn't it just in order?

Isn't just `index` a better name maybe?

Reply:
>Isn't it just in order?

They are ordered, and it's why we are setting it to the index inside the vector. But the idea is to be able to retrieve it from a ScanComponent without going through the whole vector of Components.
Or maybe you're asking if scans are ordered following the components' definition in the start of frame. In this case, the answer is no.

>Isn't just index a better name maybe?

I wanted to make it clear that it is specific to our implementation, and index might be confused with id.

Reply:
>I wanted to make it clear that it is specific to our implementation

I added it to the comment